setup.py

```diff
@@ -73,7 +73,7 @@ def package_tree(pkgroot):
                        'Programming Language :: Python :: 3',
                        ],
           platforms='any',
-          install_requires=['numpy>=1.11.3', 'scipy>=0.17.1'],
+          install_requires=['numpy>=1.11.3', 'scipy>=0.17.1', 'pysurfer'],
```

Comment:
I'm not sure to understand why this is necessary.

Reply:
It shouldn't be, PySurfer is an optional not required dependency

Reply:
Well, the thing is tha if you don't have pysurfer installed, mne just crash when you try to call this function. It does not feels like a proper behavior for an optional package, that is why I added the dependencies. I can remove this change, but at some point such exceptions for missing (optional) packages should probably be better managed (e.g., trowing a custom warning suggesting the user to install the corresponding package).

Reply:
Yes please remove it. Usually we do try to throw a custom error, we must have missed it in this function, feel free to add it along the lines of 

https://github.com/mne-tools/mne-python/blob/master/mne/viz/_3d.py#L1860-L1861

Reply:
Done with commit 9015a62